examples/base-sepolia-match/index.ts

```diff
@@ -0,0 +1,135 @@
+import {
+    type AuthConfig,
+    CHAIN_IDS,
+    type GetExternalMatchQuoteReturnType,
+} from "@renegade-fi/core";
+import { createAuthConfig } from "@renegade-fi/node";
+import { Token } from "@renegade-fi/token";
```

Comment:
Hm everything should be import-able through the `@renegade-fi/node` namespace. `@renegade-fi/core` isn't necessarily meant to be a direct dependency. I can clean this up when i migrate to the new client.